### ISSUE TITLE
Setting Project's Resolution Shrinks Text and Images

### ISSUE THREAD

**Issue by d-j-m-0** [trevorsandy/lpub3d#83]
### Subject
A project can have its resolution altered via the "Project Global Setup" dialog window. The expectation is that this would result in higher quality images being generated, that are more suitable when printing building instructions rather than displaying them on a screen.

When a resolution is set however, the size (width/height) of the images generated by LDView are smaller. Further, the text on the page is also reduced.

Moreover, although the dialog window allow a value to be specified that is suitable for use with printed material (e.g. 600 dpi), LPub3D does not use the value entered but replaces it with another value (236 in this situation).

### Environment
Version of LPub3D - 2.2.2 Revision 0 Build 848
Operating system - Windows 10 Pro (10.0.17134) , x86_64

### Configuration
N/A

### Logs
N/A

### Screenshots
N/A

### Steps to reproduce
The attached zip file contains 3 simple models; default_dpi.ldr, 236_dpi.ldr and 600_dpi.ldr.
[resolution.zip](https://github.com/trevorsandy/lpub3d/files/2355155/resolution.zip)

1) Extract the LDraw models from the zip file
2) Open LPub3D. Ensure LDView is set as the renderer.
2) Open the model default_dpi.ldr. This model uses the default LPub3D resolution.
3) Open the model 600_dpi.ldr. This model uses a (manually edited) resolution of 600 dpi i.e. suitable for use on printed material. Compare the size of the images and text with that of default_dpi.ldr
4) Open the model 236_dpi.ldr. This model uses the resolution set by LPub3D even though a value of 600 dpi is entered in the "Project Global Setup" dialog window. Compare the size of the images and text with that of default_dpi.ldr.

### Expected behaviour
The size of the text and images in the instructions should be the same across all models. The resolution of the generated images should reflect the resolution specified in each given file.

### Actual behaviour
When a higher valued resolution is specified, both the test and the images are reduced in size.

### Workaround
When using a higher valued resolution;
a) set a "scale" value for every image to increase the width and height of the generated image
b) change the font size for anything which generated text (e.g. part counts, page numbers, INSERT TEXT commands, etc)

**Comment by d-j-m-0** [comment]
Okay, a bad example for the images as they are generated the same size across all three models. The text is however reduced in size. I'll come up with another model to demonstrate the image shrinkage and post that here as an update.

David

**Comment by trevorsandy** [comment]
David - Thanks for this write-up. Indeed, I've encountered several limitations with both the renderer and the way the legacy LPub renderer commands are formulated while implementing the "True" fade feature released with version 2.3.0. In fact, I've done some work to directly access the LDView classes from within LPub3D (this is possible as all the renderers including POV-Ray are built from source when compiling LPub3D) to lay the foundation to address some of the limitations and quirkiness I've discovered. 

Cheers,
